modules/command/src/main/java/org/apache/fluo/command/FluoCommandRunner.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.fluo.command;
+
+public class FluoCommandRunner {
```

Comment:
This class could probably be combined with the `FluoProgram` class. It might be better to have the logic of both of these classes in the same place. I'm interested to hear others opinions on it.

From the Command Pattern I tried to make this the Invoker class.

Reply:
I am in favor of moving it to FluoProgram.  If you inlined the static method runFluoCommand then all calls to System.exit may be in FluoProgram.main.